plugin/src/main/java/org/opensearch/ml/task/MLPredictTaskRunner.java

```diff
@@ -458,13 +502,15 @@ private void runPredict(
                                 }
                             } else {
                                 handleAsyncMLTaskComplete(mlTask);
-                                mlModelManager.trackPredictDuration(modelId, startTime);
```

Comment:
`mlModelManager.trackPredictDuration(modelId, startTime)` will do this
```
modelCacheHelper.addModelInferenceDuration(modelId, durationInMs);
```

In the new code, don't see `addModelInferenceDuration` in `recordPredictMetrics` , that seems break BWC ?